client/src/components/HandMarkedPaperBallot.tsx

```diff
@@ -33,6 +34,38 @@ import QRCode from './QRCode'
 import Prose from './Prose'
 import Text from './Text'
 
+const qrCodeTargetClassName = "qr-code-target"
```

Comment:
I'm surprised prettier isn't making these single quotes.

Reply:
🤦 Just realizing that I have to install all the VSCode extensions in the VM for them to work. Will commit linted changes.